usr/sbin/truenas-install

```diff
@@ -388,6 +388,8 @@ create_partitions()
     local _disk="$1"
     local _sector_size=$(get_disk_logical_sector_size /dev/${_disk})
     local _alignment_multiple=4096
+    local _legacymsg="Legacy boot is detected. Do you want to be able boot in EFI mode as well vs more compatible legacy?"
```

Comment:
We need to get a better sentence here. @Mrt134 can you help?

Reply:
In most cases SCALE set for EFI boot should also be able to boot legacy.  So may be "Allow EFI boot" instead and may be "requires legacy BIOS boot workaround"?

Reply:
I have updated to "Allow EFI boot? Enter Yes for systems with newer components such as NVMe devices. Enter No when system hardware requires legacy BIOS boot workaround."
![image](https://github.com/truenas/truenas-installer/assets/97105857/e5bba656-0863-49bb-b121-2d6236202e5b)